src/init/mod.rs

```diff
@@ -94,6 +94,13 @@ fi"#,
             );
             Some(script)
         }
+        Some("powershell") => {
+            let script = format!(
+                "Invoke-Expression (@(&\"{}\" init powershell --print-full-init) -join \"`n\")",
```

Comment:
I'm not a Powershell user, so I'm not super familiar with what's going on here, specifically the symbols (`(@(&`), nor why ``-join \"`n\"`` is needed.

Would the average Powershell user understand this, or should comments be added?

Reply:
I've added explanations to this module and the powershell script.